starters/next-wordpress-starter/pages/pages/index.jsx

```diff
@@ -15,7 +16,7 @@ export default function PageListTemplate({ menuItems, pages }) {
         description="Generated by create next app."
       />
       <PageHeader title="Pages" />
-      <section>
+      <section className="has-luminous-vivid-amber-background-color">
```

Comment:
Left over from testing?

Reply:
Oh yes, I forgot to revert this, thanks for the catch